test/integration/test_grpc_service.py

```diff
@@ -42,3 +42,25 @@ def test_grpc_service_stress(http_test_server_fixture):
   counters = http_test_server_fixture.getNighthawkCounterMapFromJson(parsed_json)
   assertGreaterEqual(counters["benchmark.http_2xx"], 5000)
   assertEqual(counters["requestsource.internal.upstream_rq_200"], 4)
+
+
+def run_service_with_args(args):
+  return run_binary_with_args("nighthawk_service", args)
+
+
+def test_grpc_service_help():
+  (exit_code, output) = run_service_with_args("--help")
+  assert (exit_code == 0)
```

Comment:
Since these are tests, we should likely use self.assertTrue instead of assert. I think assert just terminates the execution while self.assertFoo correctly records the test failure using the test framework.

Reply:
Well, we're using `pytest`, I think this ought te be ok. quoting from the docs:
```
Due to pytest’s detailed assertion introspection, only plain assert statements are used. 
```

Posting the output of a sample failure (introduced on purpose):

```
    def test_output_transform_help():
      (exit_code, output) = _run_output_transform_with_args("--help")
>     assert (exit_code == 1)
E     assert 0 == 1
E       -0
E       +1

exit_code  = 0
output     = ('WARNING: Perftools heap leak checker is active -- Performance may suffer\n'
 '\n'
 'USAGE: \n'
.... <snip> ...
```


[1] https://docs.pytest.org/en/stable/

Reply:
Having said that, we do have some helpers in `utilities.py`, like `assertEqual`. Switched to use that, for consistency with the rest of the test code.